css/portal-dashboard/class-nav.less

```diff
@@ -10,9 +10,13 @@
 
   .chooseClass {
     margin: 20px 0 15px 19px;
+    width: 200px;
+    height: 30px;
```

Comment:
I think you can get away with not adding these.  The width here is actually constraining the width of the custom select header so it means the custom select needs a parent container to control the width.  I think it's better to be able to place a custom component anywhere without having to add a styled parent to control it's layout.